lib/zendesk/client/operation.ex

```diff
@@ -5,15 +5,16 @@ defmodule Zendesk.Client.Operation do
   alias Zendesk.Client.Parser
   alias __MODULE__
 
-  defstruct path: "/", type: :get, parser: &Parser.default_parse/1, params: []
+  defstruct path: "/", type: :get, parser: &Parser.default_parse/1, params: [], body: nil
```

Comment:
👍 
I could engage in more HTTP bike-shedding an suggest that we could send a body with `get`s, but it'd be pointless and now I'm being super annoying. Apologies. Please don't randomly select me to host the next Happy Hour.

Reply:
You're right!  But I don't think there are any `get` endpoints in the API that accept a body, so think we should be good 😄